sippy-ng/src/component_readiness/CompReadyVars.js

```diff
@@ -240,6 +264,101 @@ export const CompReadyVarsProvider = ({ children }) => {
+        'Exclude Networks': defaultExcludeNetworksCheckedItems,
+        'Exclude Clouds': defaultExcludeCloudsCheckedItems,
+        'Exclude Upgrades': defaultExcludeUpgradesCheckedItems,
+        'Exclude Variants': defaultExcludeVariantsCheckedItems,
+        Confidence: defaultConfidenceParam,
+        Pity: defaultPityParam,
+        'Min Fail': defaultMinFailParam,
+        'Ignore Missing': defaultIgnoreMissingParam,
+        'Ignore Disruption': defaultIgnoreDisruptionParam,
+      },
+    },
+    ARM64: {
+      config: {
+        help: 'Default view but only for amd64',
+        'Group By': defaultGroupByCheckedItems,
+        'Exclude Arches': ['amd64', 'ppc64le', 's390x', 'heterogeneous'],
+        'Exclude Networks': defaultExcludeNetworksCheckedItems,
+        'Exclude Clouds': defaultExcludeCloudsCheckedItems,
+        'Exclude Upgrades': defaultExcludeUpgradesCheckedItems,
+        'Exclude Variants': defaultExcludeVariantsCheckedItems,
+        Confidence: defaultConfidenceParam,
+        Pity: defaultPityParam,
+        'Min Fail': defaultMinFailParam,
+        'Ignore Missing': defaultIgnoreMissingParam,
+        'Ignore Disruption': defaultIgnoreDisruptionParam,
+      },
+    },
+    Heterogeneous: {
+      config: {
+        help: 'Default view but only for heterogeneous architecture',
+        'Group By': defaultGroupByCheckedItems,
+        'Exclude Arches': ['amd64', 'arm64', 'ppc64le', 's390x'],
+        'Exclude Networks': defaultExcludeNetworksCheckedItems,
+        'Exclude Clouds': defaultExcludeCloudsCheckedItems,
+        'Exclude Upgrades': defaultExcludeUpgradesCheckedItems,
+        'Exclude Variants': defaultExcludeVariantsCheckedItems,
+        Confidence: defaultConfidenceParam,
+        Pity: defaultPityParam,
+        'Min Fail': defaultMinFailParam,
+        'Ignore Missing': defaultIgnoreMissingParam,
+        'Ignore Disruption': defaultIgnoreDisruptionParam,
+      },
+    },
+  }
+
+  // Save user views to browser local storage; if empty, we're saving {}.
+  // You can view the storage by "inspecting the page" (F12 on some browsers), and
+  // going to the Application tab.
+  const saveViewsToLocal = (views) => {
+    const viewsToSave = Object.entries(views).reduce((acc, [key, value]) => {
+      if (value.config.class === 'user') {
+        acc[key] = value
+      }
+      return acc
+    }, {})
+    localStorage.setItem('savedViews', JSON.stringify(viewsToSave))
+  }
+
+  // See if user has saved views.
+  const loadViewsFromBrowser = () => {
+    // If there is data in the local storage and it has keys, then merge it with the
+    // initial views.  Otherwise, just use the initial views.
+    const savedViews = localStorage.getItem('savedViews')
+    if (savedViews && Object.keys(JSON.parse(savedViews)).length > 0) {
+      const mergedViews = { ...initialViews, ...JSON.parse(savedViews) }
+      return mergedViews
+    } else {
+      return initialViews
+    }
+  }
```

Comment:
Was just using bookmarks discussed? Why would we save to local storage when we could fully qualify a URL? I forsee problems with what people have locally in the future, whereas bookmarks gives us a path to reporting a problem with a URL, that might be more tricky if we hit some bug due to local storage.

Reply:
ah, bookmark is a browser bookmark and not the bookmark widget in the sippy UI.